src/Pmi/Service/SiteSyncService.php

```diff
@@ -175,14 +195,18 @@ public function syncOrganizations()
                 $organizationsMap[$organization->id] = $organization->displayName;
             }
         }
+        if (empty($organizationsMap)) {
+            throw new \Exception('No organizations found');
+        }
         $organizationsRepository = $this->em->getRepository('organizations');
         $organizationsRepository->wrapInTransaction(function() use ($organizationsRepository, $organizationsMap) {
             $organizationsRepository->truncate();
             foreach ($organizationsMap as $id => $name) {
-                $organizationsRepository->insert([
+                $organizationId = $organizationsRepository->insert([
                     'id' => $id,
                     'name' => $name
                 ]);
+                $this->app->log(Log::ORGANIZATION_ADD, $organizationId);
```

Comment:
Same suggestion here

```suggestion
                $this->app->log(Log::ORGANIZATION_ADD, $id);
```